ReleaseProcess.md

```diff
@@ -18,3 +18,24 @@
    1. `cd /montagu/deploy`
    1. `./deploy.py --docker-hub`
 7. Use `RELEASE_LOG.md` to know which tickets to update to the 'Deployed' status
+
+## Deploying to UAT
+
+1. `ssh support.montagu`
```

Comment:
I usually combine step 1 & 2 with `ssh vagrant@support.montagu`, which I think is a touch cleaner (`sudo` always being ever so slightly weird) - is that worth mentioning as an approach here?